requirements.txt

```diff
@@ -1,4 +1,4 @@
 numpy>=1.18.0
 pandas>=0.25.2
-scikit-learn>=0.22.1
+scikit-learn>=1.0.0
```

Comment:
Thoughts @adrinjalali ? This is related to #1288

Reply:
oh yeah, I'd even do 1.1, we're going to release 1.4.